cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -184,18 +179,26 @@ def is_flow_cell_directory_valid(self, flow_cell_directory: Path) -> bool:
     def is_demultiplexing_complete(self, flow_cell_directory: Path) -> bool:
         return Path(flow_cell_directory, DemultiplexingDirsAndFiles.DEMUX_COMPLETE).exists()
 
-    def update_sample_read_counts(self, sample_internal_ids: List[str]) -> None:
+    def update_sample_read_counts(self, flow_cell_data: FlowCellDirectoryData) -> None:
         """Update samples in status db with the sum of all read counts for the sample in the sequencing metrics table."""
+
+        q30_threshold: int = FLOWCELL_Q30_THRESHOLD[flow_cell_data.sequencer_type]
+
+        sample_internal_ids: List[str] = self.get_sample_ids_from_sample_sheet(
+            flow_cell_data=flow_cell_data
+        )
+
         for sample_id in sample_internal_ids:
-            self.update_single_sample_read_count(sample_id)
+            self.update_single_sample_read_count(sample_id=sample_id, q30_threshold=q30_threshold)
```

Comment:
```suggestion
            self.update_single_sample_read_count(sample_id=sample_id, q30_threshold=FLOWCELL_Q30_THRESHOLD[flow_cell_data.sequencer_type])
```

Reply:
I think this is less readable and prefer keeping it in a variable as above, alternatively extracting it to a function like
```python
def get_q30_threshold(sequencer_type: str):
    return FLOWCELL_Q30_THRESHOLD[sequencer_type]
```